src/content/1.7/modules/testing/resources.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Testing Resources
+weight: 10
+---
+
+# Testing Resources
+
+## Docker images
+
+Docker isolate your application in containers, making their isolation and deployment easier.
+If you're confident with using it, we created several images for easier deployment of PrestaShop:
+
+* https://hub.docker.com/r/prestashop/prestashop/, in which all PrestaShop releases between 1.4 and 1.7 can be found. Covers most needs and is perfect for a quick start.
+* https://hub.docker.com/r/prestashop/prestashop-git/, containing the git repository content with different PHP version. For advanced users.
+* https://hub.docker.com/r/prestashop/base/, containing the required stack to run PrestaShop, without the core files. Recommended if you already have the shop files on your disk.
+
+Please note we try to follow best practices, and the MySQL is NOT provided with these images. You have to deploy your own server in a dedicated container.
+
+
+When your environment is ready, it's time to configure it before running tests.
+
+## Module management in command line
+
+> From PrestaShop {{< minver v="1.7.2" >}}
+
+Managing your modules was only possible from the administration inferface. Continuous integration of modules may require preliminary steps like their installation, that's why we added a simple way to manage them in command line.
+
+```bash
+php bin/console prestashop:module --help
+```
+
+```term
+Usage:
+  prestashop:module <action> <module name> [<file path>]
+
+Arguments:
+  action                Action to execute (Allowed actions: install / uninstall / enable / disable / enable_mobile / disable_mobile / reset / upgrade / configure).
+  module name           Module on which the action will be executed
+  file path             YML file path for configuration
+
+```
+
+{{% notice note %}}
+Use `php app/console` instead of `php bin/console` for versions prior to {{< minver v="1.7.4" >}}
+{{% /notice %}}
+
+### Basic actions
+
+The basic actions are the same as you can see on the module list and only require the module name.
+
+* install
+* uninstall
+* enable
+* disable
+* enable_mobile
+* disable_mobile
+* reset
+* upgrade
+
+Example:
+```term
+php bin/console prestashop:module install paypal
+```
+
+{{% notice note %}}
+Use `php app/console` instead of `php bin/console` for versions prior {{< minver v="1.7.4" >}}
+{{% /notice %}}
+
+#### Multishop
+In case you want to run your action for a specific shop or shop group, you have two optionnal parameters ``--id_shop=<id>`` and ``--id_shop_group=<id>`` which modify the current shop context before calling the actual command.
```

Comment:
```suggestion
In case you want to run your action for a specific shop or shop group, you have two optionnal parameters ``--id_shop=<id>`` and ``--id_shop_group=<id>`` which modifies the current shop context before calling the actual command.
```